src/main/java/uk/ac/ebi/eva/commons/mongodb/services/VariantSourceService.java

```diff
@@ -20,7 +20,7 @@
         return convert(repository.findAll());
     }
 
-    private List<VariantSource> convert(List<VariantSourceMongo> variantSources) {
```

Comment:
Creating a projection from the interface might just make the code unnecessarily more complex, and would it even make sense to do this, having yet another model of a variant source seems unnecessary.